README.md

```diff
@@ -30,8 +30,23 @@ SelfCare Header component
 | Prop | Type | Mandatory | Description |
 |------|------|-----------|-------------|
 | withSecondHeader | Y | boolean | if true, it will render an other toolbar under the Header |
+| productsList| Y |Array`ProductEntity`| If withSecondHeader is true, this component will be rendered at the end of the secondary toolbar |
+| selectedPartyId | N | the party selected |
+| selectedProductId | N | the product id selected |
+| partyList | N | Array`PartyEntity` | the parties list |
+| loggedUser | Y | `JwtUser | false` | shows if there is a logged user |
+| assistanceEmail | N | string | the email to which the assistance button will ask to send an email |
+| onSelectedProduct | N | (product: ProductSwitchItem) => void | the function to be invoked when the party was selected |
+| onSelectedParty | N | (exitAction: () => void) => void | the function to be invoked when the party was selected | 
+| onAssistanceClick | N | the function to be invoked to redirect the user to the assistance page | 

```

Comment:
fix the type of the parameter and change the description to "The function invoked when clicking on the assistance button"